lib/suspenders/app_builder.rb

```diff
@@ -196,13 +196,11 @@ def configure_i18n_for_missing_translations
     end
 
     def configure_i18n_tasks
```

Comment:
seems like we shouldn't include the `yml` file if we are removing the spec file that uses it?